src/resources/assets/js/components/dismissImageGridImage.vue

```diff
@@ -3,18 +3,40 @@
         <div v-if="selectable" class="image-icon">
             <i class="fas" :class="iconClass"></i>
         </div>
-        <img @click="toggleSelect" :src="srcUrl" @error="showEmptyImage">
+        <div class="overlay svg">
+            <img v-if="this.svg === null" :src="srcUrl" @error="showEmptyImage" @click="toggleSelect">
+            <div v-else @click="toggleSelect">
```

Comment:
This looks like it should rather be:

```
<img @click :src @error>
<div v-if="this.svg !== null" class="overlay">
   <span v-html="this.svg"></span>
   ...
</div>
```

This uses less nesting of HTML elements, does not duplicate the `<img>` and you could even put a `pointer-events: none` in the CSS of the `overlay` so the mouse event on the `<img>` still works with the overlay.